moveit_kinematics/ikfast_kinematics_plugin/templates/ikfast61_moveit_plugin_template.cpp

```diff
@@ -180,7 +180,7 @@ class IKFastKinematicsPlugin : public kinematics::KinematicsBase
    * @return True if a valid solution was found, false otherwise
    */
 
-  // Returns the first IK solution that is within joint limits, this is called by get_ik() service
+  // Returns the IK solution that is within joint limits closest to ik_seed_state, this is called by get_ik() service
```

Comment:
I know this is legacy code that you aren't involved in, but could you remove the comment ", this is called by get_ik() service" - that is referring to some really old code that no longer exists